### ISSUE TITLE
[EN][Dungeon-WotLK] Gundrak

### ISSUE THREAD

**Issue by Tj20** [Atlantiss/BugTracker#2789]
http://www.wowhead.com/zone=4416/gundrak

### [Gal'darah](http://www.wowhead.com/npc=29306/galdarah#encounter)
- In our 46 sec long fight, Gal'darah did only use melee attacks and [Whirling Slash](http://www.wowhead.com/spell=55250) ([combat log](http://pastebin.com/PEsY9QtV)).
- He didn't have a phase 2 (transformation into a rhino) and didn't cast his other abilites [Stampede](http://www.wowhead.com/spell=155247), [Puncture](http://www.wowhead.com/spell=55276), [Impaling Charge](http://www.wowhead.com/spell=54956), [Enrage](http://www.wowhead.com/spell=22428), [Stomp](http://www.wowhead.com/spell=55292)

### Trash Mobs
- The trash before the endboss is missing: 4 Drakkari Rhinos([1](http://www.wowhead.com/npc=29931/drakkari-rhino),[2](http://www.wowhead.com/npc=29838/drakkari-rhino)) and the trolls around them ([Drakkari Raiders](http://www.wowhead.com/npc=29982), [Drakkari Battle Riders](http://www.wowhead.com/npc=29836)), [video](https://youtu.be/69E4zeHtGgE?t=7m35s)

![pic](http://i.imgur.com/vrvDkPR.jpg)

### Entrances & Quests
![pic](http://i.imgur.com/yL9QKiq.jpg)

**Observed Behavior:**  
The instance has 2 entrances with 2 quest givers: [Tol'mar](http://www.wowhead.com/npc=55738/tolmar) and [Chronicler Bah'Kini](http://www.wowhead.com/npc=55737/chronicler-bahkini). Both quest givers give 3 quests with the same title and the same rewards: *For Posterity*([T](http://www.wowhead.com/quest=29844),[B](http://www.wowhead.com/quest=29840)), *Gal'darah Must Pay*([T](http://www.wowhead.com/quest=29834),[B](http://www.wowhead.com/quest=29835)), and *One of a Kind*([T](http://www.wowhead.com/quest=29839),[B](http://www.wowhead.com/quest=29838)). You can complete all 6 quests in 2 Gundrak runs.

![our server](http://i.imgur.com/PPwcMYz.jpg)

The dungeon finder apparently places your group at a random entrance. I did the dungeon twice: first we were spawned on the left side. Next run, we were placed on the right side.

**Expected Behavior:**  
Hmm...On all my years on retail as alliance I was only spawned at the right entrance and didn't do quests twice. So I thought: Maybe the left side is for horde groups and is a horde quest giver? After looking at some horde POV Cata videos, it seems to me that the DF also placed them on the right side so that they started with the snake boss, too. ([video](https://youtu.be/ZcjgN_bBtGo?t=30s))

As for the quests, I think both quest givers should give the same quests, see this [video](https://www.youtube.com/watch?v=GRIdH_RnzLA): The horde group starts right, gets the 3 quests from Tol'mar, and when they reach the left side of the dungeon with 2 quests finished you see the yellow questionmark on the map meaning that he could turn the quests in at Bah'Kini also.

![other](http://i.imgur.com/02pLdxm.jpg)

**Comment by Kejmil** [comment]
yeah, it should be the same quests or they should exclude each other, don't remember which one was it
